2.14/2.14.4/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.14.4
+Сообщение первое: "Введите первое число от 0 до 40" (например ввели 12)
+Сообщение второе: "Введите второе число от 0 до 40" (например ввели 21)
+Потом появляется алерт: "Сумма этих чисел равна тридцать три" (для нашего примера получился ответ 33)
+Специфика в том, чтобы в алерте выводить числа текстом.
+Предусмотреть текст для чисел в диапазоне 0 - 99.
+<script>
+	const userFirstNum = +prompt('Введите первое число от 0 до 40');
+	const userSecondNum = +prompt('Введите второе число от 0 до 40');
+
+	let sum = userFirstNum + userSecondNum;
+
+	if (sum >= 0 && sum < 10) {
+		switch (sum) {
+			case 0:
+				sum = 'ноль';
+				break;
+			case 1:
+				sum = 'один';
+				break;
+			case 2:
+				sum = 'два';
+				break;
+			case 3:
+				sum = 'три';
+				break;
+			case 4:
+				sum = 'четыре';
+				break;
+			case 5:
+				sum = 'пять';
+				break;
+			case 6:
+				sum = 'шесть';
+				break;
+			case 7:
+				sum = 'семь';
+				break;
+			case 8:
+				sum = 'восемь';
+				break;
+			case 9:
+				sum = 'девять';
+				break;
+		}
+	}
+	if (sum > 9 && sum < 20) {
+		switch (sum) {
+			case 10:
+				sum = 'десять';
+				break;
+			case 11:
+				sum = 'одинадцать';
+				break;
+			case 12:
+				sum = 'двенадцать';
+				break;
+			case 13:
+				sum = 'тринадцать';
+				break;
+			case 14:
+				sum = 'четырнадцать';
+				break;
+			case 15:
+				sum = 'пятнадцать';
+				break;
+			case 16:
+				sum = 'шестнадцать';
+				break;
+			case 17:
+				sum = 'семнадцать';
+				break;
+			case 18:
+				sum = 'восемнадцать';
+				break;
+			case 19:
+				sum = 'девятнадцать';
+				break;
+		}
+	}
+	if (sum > 19) {
+		let firstNum = Math.floor(sum / 10);
+		let secondNum = sum % 10;
+
+		if (secondNum === 0) {
```

Comment:
do you need this ˋifˋ here?
and this is a bit strange that you are checking secondNum, then firstNum and then secondNum again